.goreleaser.yml

```diff
@@ -5,6 +5,7 @@ builds:
     binary: subnet-evm
     flags:
       - -v
+      - -ldflags "-X github.com/ava-labs/subnet-evm/plugin/evm.Version=v0.2.5"
```

Comment:
Note: a further improvement would be to set this by passing this in as an environment variable to the GH Action.

Reply:
done